### PR TITLE
Added troubleshooting topic for appliance welcome page error

### LINKED ISSUE #1282: Add troubleshooting doc about encrypted private key when providing TLS cert
**VIC Product version:**

OVA version vic-v1.3.0-f8cc7317.ova 

**Deployment details:**

OVA import to vSphere 6.5. Fist boot up shows EXT4-fs (sda2): couldnt't mount as ext3 due to feature incompatibilities

**Steps to reproduce:**

OVA import to vSphere Host and power up

**Actual behavior:**

VIC Management dosen't run

**Expected behavior:**

**Logs:**

**Additional details as necessary:**

### PR DESCRIPTION
**Pull request by vidya-v**
Added a new topic under [Troubleshoot vSphere Integrated Containers Appliance Deployment](https://vmware.github.io/vic-product/assets/files/html/1.4/vic_vsphere_admin/ts_deploy_appliance.html)

Fixes #1282 

Hi @stuclem , can you please review this PR? Thanks!

### CODE REVIEW

**Comment by vidya-v** [comment]
Hi @zjs , can you please review this troubleshooting topic? Thanks!

File: docs/user_doc/vic_vsphere_admin/vicappliance_welcomepage_error.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#Appliance Welcome Page Error#
```
Comment (zjs) [inline docs/user_doc/vic_vsphere_admin/vicappliance_welcomepage_error.md]: Most headings on this page don't seem to [render properly](https://github.com/vmware/vic-product/blob/9399c82f70f2256c7d63d2a8aa30c71b63b8f0e3/docs/user_doc/vic_vsphere_admin/vicappliance_welcomepage_error.md). I think they need a space between the `#` character(s) and the first letter.

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/vicappliance_welcomepage_error.md]: Thanks @zjs , I have corrected the headings.

**Comment by stuclem** [comment]
@vidya-v I think that we can safely merge this now. Thanks!

**Pull request merged**